releases.json

```diff
@@ -125,6 +125,14 @@
       "1.1.5-1"
     ]
   },
+  "centurion": {
+    "dependency_names": [
+      "centurion"
+    ],
+    "versions": [
+      "7.2.0"
```

Comment:
This version needs to include both the upstream release number and the WrapDB local build number (starting with "1") separated by a hyphen.

See https://github.com/mesonbuild/wrapdb#how-to-contribute-new-wraps as well as the style used by other entries in this file.

Reply:
Ah, thank you, just figured it out!